### PR TITLE
[DOCS-202] Details for Mobile Methodology

### PR DESCRIPTION
**Pull request by mjang-cobalt**
## Changelog

 Approved by CSM over internal chat. Zendesk article already updated to point to product docs. Ready for your review, @ana-dashuk-cobalt 

### Added

n/a

### Updated

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| [Mobile Methodologies] | [[Link](https://deploy-preview-233--cobalt-docs.netlify.app/platform-deep-dive/pentests/pentest-process/methodologies/mobile/)] | [Updated per CSM request] |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 1]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+assess the archive as well as the local file.
+
+Our pentesters focus on:
+
+- Communication channels
+- Traffic that the application exchanges with external endpoints
+- Inter-Process Communication (IPC)
+
+Our pentesters may also reverse-engineer the application for insight, and to
+try to access sensitive data.
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:87]:
```suggestion
Our pentesters may also reverse engineer the application for insight and try to access sensitive data.
```

https://www.merriam-webster.com/dictionary/reverse%20engineer

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 2]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+
+For backend tests, they use [API Pentest Methodologies](platform-deep-dive/pentests/pentest-process/methodologies/api-methodologies/).
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:89]:
```suggestion
For backend tests, they use [API Pentest Methodologies](/platform-deep-dive/pentests/pentest-process/methodologies/api-methodologies/).
```

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 3]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+
+- Reach and scan the targets
+- Test the functionality of the application
+
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:69]: @mjang-cobalt Can we set up a shortcode for this? I see that this part is repeated several times.

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 4]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+
+{{% /expand %}}
+
+{{% expand "Report, Triage, and Restest" %}}
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:122]:
```suggestion
{{% expand "Report, Triage, and Retest" %}}
```

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 5]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+</br>
+
+Pentesters report and triage all findings as they work. We share details through our online platform. 
+We encourage you to address critical findings as we discover them. Our pentesters can retest, based
+on your [PtaaS Tier](/platform-deep-dive/credits/ptaas-tiers/).
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:127]:
```suggestion
Pentesters report and triage all findings as they work. We share details through our online platform. 
We encourage you to address critical findings as we discover them. Our pentesters can retest findings, based
on your [PtaaS tier](/platform-deep-dive/credits/ptaas-tiers/).
```

> We share details through our online platform.

Should we specify where exactly customers can view findings?

> based on your [PtaaS Tier](/platform-deep-dive/credits/ptaas-tiers/)

I'm not sure we need to add a link to the new article about PtaaS tiers. I think that in most cases we should still redirect users to https://www.cobalt.io/pentest-pricing. What do you think?

**Review by ana-dashuk-cobalt — APPROVED** [review]
@mjang-cobalt  Approving with several comments.

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 6]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
 full coverage:
 
-- Reconnaissance
-  - Share the mobile application files
-    - Android: .apk
-    - iOS: .ipa
-- Automated and Manual Testing
-- Exploit Discovered Vulnerabilities
-- Report, triage, and retest
+Select the <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> key for more information on each step.
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:47]: @mjang-cobalt I think there should be a way to use our #primary color. I experimented with HTML but didn't get the result we need.

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 7]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
+
+{{% /expand %}}
+
+{{% expand "Automated and Manual Testing" %}}
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:73]:
```suggestion
{{% expand "Automated and manual testing" %}}
```

Sentence case for consistency.

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md [hunk 8]
```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
...
+Our pentesters may use scanning tools such as:
+
+- MobSF
+- Frida
+- Apktool
+- Dex2Jar
+
+{{% /expand %}}
+
+{{% expand "Exploit Discovered Vulnerabilities" %}}
```
Comment (ana-dashuk-cobalt) [inline content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:100]:
```suggestion
{{% expand "Exploit discovered vulnerabilities" %}}
```

Comment (mjang-cobalt) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:100]: Oh duh, thanks. It is _not_ a title!

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md (hunk 6 above)
Comment (mjang-cobalt) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:47]: I _think_ the problem might be related to my use of Font Awesome icons (`fas`)

File: content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md (hunk 3 above)
Comment (mjang-cobalt) [reply content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md:69]: There are a number of sections in all of these where we should set up short codes. And while I should start now, I was having trouble thinking, "what should I name the file"... But your question is making me think again... and I realize they're all the same in this particular methodology. ding!

**Pull request merged**